### DISCUSSION TITLE
When can I use QPopupProxy in NiceGUI?

### DISCUSSION THREAD

**Discussion by hu76589** [zauberzeug/nicegui#1628] [Q&A]
### Description

Do you have any plans to add QPopupProxy to nicegui？
Or maybe there's a way to use it now？
tks

**Comment by falkoschindler** [comment]
You can use the generic `ui.element` for Quasar elements that aren't natively supported yet:
```py
with ui.button('Show banner'):
    with ui.element('q-popup-proxy'):
        with ui.element('q-banner'):
            ui.label('This is a banner')
```
